bundler/lib/dependabot/bundler/update_checker/latest_version_finder.rb

```diff
@@ -55,9 +56,10 @@ def fetch_lowest_security_fix_version
 
           relevant_versions = dependency_source.versions
           relevant_versions = filter_prerelease_versions(relevant_versions)
-          relevant_versions = filter_vulnerable_versions(relevant_versions)
           relevant_versions = filter_ignored_versions(relevant_versions)
           relevant_versions = filter_lower_versions(relevant_versions)
+          relevant_versions = Dependabot::UpdateCheckers::VersionFilters.filter_vulnerable_versions(relevant_versions,
+                                                                                                    security_advisories)
```

Comment:
@Nishnha quick flag, I think we should revert the order of these filters across all ecosystems to be `filter_vulnerable_versions` then `filter_ignored_versions`. I think we've got spotty test coverage covering this order so might have just failed in a couple.

Reply:
Since this PR is already merged in I opened a new one to address the issue 
https://github.com/dependabot/dependabot-core/pull/3962

Reply:
Thanks appreciate the quick fix!